### ISSUE TITLE
Video stopped playing

### ISSUE THREAD

**Issue by UlyaKotova** [WWBN/AVideo#7549]
Hello!
At september 2022 I installed site and encoder on one server on ubuntu 20.04. Both worked properly, videos are being uploaded, converted and can be viewed. All worked perfectly. Before today ;(

But today we found some issues with video player - cant view no one video. Player doesnt work (when we push the button "play" on site - nothing changed). Here some screenshots and info from log:
![Problem1](https://user-images.githubusercontent.com/122457698/211824351-b4a7ed27-23ce-4fca-bd93-36bbb3fdb6f7.jpg)
![изображение](https://user-images.githubusercontent.com/122457698/211824544-44881917-1a7a-453a-a5b6-23aa140c8b8b.png)
![изображение](https://user-images.githubusercontent.com/122457698/211827678-34d6fd51-f30a-4e5d-bdd7-f90f67d3d22b.png)
![изображение](https://user-images.githubusercontent.com/122457698/211827905-e4b29007-be81-4127-850c-b759e26e6f6e.png)
![изображение](https://user-images.githubusercontent.com/122457698/211826510-446a3ab5-b603-4666-a564-0a33776ff824.png)
![изображение](https://user-images.githubusercontent.com/122457698/211826633-8d17d152-5241-4b41-9aae-c7eaae6c5e65.png)
![изображение](https://user-images.githubusercontent.com/122457698/211826723-ac8dad27-8e0c-4dc0-98af-b4b1cfb0813f.png)

We updated version from 12.1 to 12.3 but have same problems.
Also we use videos on our Moodle education site - shared link in <iframe> tag - same problems, video not available for playing (different player).

Our media library not available from outside (Internet) - just LAN.

Cant understand what happend and how i can fix it. If anyone can help - will be appreciate.

**Comment by UlyaKotova** [comment]
Thanks you, [DanielnetoDotCom](https://github.com/DanielnetoDotCom) for so fastering answer.
Can i ask you to help me again - i checked video file exists! but now understand second part - how check the js console ? ><

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/wiki/How-to-find-errors-on-AVideo-Platform#javascript-errors

https://github.com/WWBN/AVideo/wiki/Using-Your-Browser-Console-to-Diagnose-JavaScript-Errors

**Comment by UlyaKotova** [comment]
Hm there some interesting ;) mb can help you:

![изображение](https://user-images.githubusercontent.com/122457698/211832399-4ef5bb0a-9d0b-41d1-9d31-d644e73227fe.png)

**Comment by DanielnetoDotCom** [comment]
I am not sure if you really find the mp4 file in your videos folder

your apache is saying the video not been found

<img width="987" alt="image" src="https://user-images.githubusercontent.com/18009293/211836936-053e7349-2dd5-40dc-83f4-b7e1e6283b69.png">

**Comment by UlyaKotova** [comment]
Yes now i understand - i have only on site admin acess (for service Avideo), i need to check on Linus server's console (linux -->etc.), right?
The problem is - im not a system administrator for real server, okay i'll try to ask help or root password ;))

Thanks you again, i'll be back tomorrow ;)))

**Comment by UlyaKotova** [comment]
right now uploaded a new file (via encoder) - all good no errors but still can play on site
![image](https://user-images.githubusercontent.com/122457698/211843547-9e517b33-8f24-457b-9daa-88b40c027480.png)
![image](https://user-images.githubusercontent.com/122457698/211843932-263e702b-9221-4422-9c5f-1fd0c4b30f40.png)

okay thanks you very much i'll try to ask our Admins for help with server.

**Comment by DanielnetoDotCom** [comment]
you need to check the avideo log and the encoder log

also confirm if the file is there (in the server directory)

**Comment by UlyaKotova** [comment]
Thanks you, [DanielnetoDotCom](https://github.com/DanielnetoDotCom) for answers.
We found errors and fixed issue (with my local Admin).
Problem was here:
![image](https://user-images.githubusercontent.com/122457698/212049565-d65c3e86-f4c2-4dd2-a9fc-34d18b989c64.png)
incorrect string in config file /etc/apache2/sites-available/**avideo.conf** 
    <Directory /var/www/AVideo>
       DirectoryIndex index.php
       Options +FollowSymLinks
       AllowOverride All
       Require all granted
       XSendFile on
       **XSendFilePath /var/www/html/AVideo/** - here was incorrect path to video files --> we changed.
       Order Allow,Deny
       Allow from All
    </Directory>

Thanks for the cool service AVideo and simple media hosting, we are discussing with local management purchase of a corporate version AVideo.
You did a really cool project, we wish you to fix all the bugs and keep developing the project.
Thanks you again for support.

**Comment by DanielnetoDotCom** [comment]
that is really important to be fixed.

it is something you must configure on your server-side NOT in the AVideo code.

if it was wrong it was a mistake from your server Admin
